tests/cucumber/steps/steps.js

```diff
@@ -426,16 +379,16 @@ module.exports = function getSteps(options) {
       this.rekey = await this.kcl.generateKey(this.handle);
       this.rekey = this.rekey.address;
       // Fund the rekey address with some Algos
-      const sp = await this.acl.getTransactionParams();
+      const sp = await this.v2Client.getTransactionParams().do();
       if (sp.firstRound === 0) sp.firstRound = 1;
       const fundingTxnArgs = {
         from: this.accounts[0],
         to: this.rekey,
         amount: DEV_MODE_INITIAL_MICROALGOS,
         fee: sp.fee,
-        firstRound: sp.lastRound + 1,
-        lastRound: sp.lastRound + 1000,
-        genesisHash: sp.genesishashb64,
+        firstRound: sp.firstRound,
+        lastRound: sp.lastRound,
```

Comment:
The change looks interesting, looks like this pattern is everywhere in js sdk, that first round used to be `lastRound + 1`, and `lastRound` is `lastRound + 1000`. Is it because of suggested params change?

Reply:
Yeah, I can't comment exactly on the old code, but the v1 `getTransactionParams()` does not natively have a `firstRound` field. Instead, the `suggestParams` in v1 aliases the `firstRound` field to `lastround` and the `lastRound` becomes `lastRound+1000`:
https://github.com/algorand/js-algorand-sdk/blob/6fdb8720633a874662b382b7ed29736f2dce1919/src/client/algod.js#L336

In v2, `firstRound` is there: https://github.com/algorand/js-algorand-sdk/blob/b780daa21485fb059fee76177eb0dfec18cea762/src/types/transactions/base.ts#L100

Reply:
that is... quite surprising, but the change is making sense to me